### PR TITLE
Update installation instruction for Gradle version

### PR DESCRIPTION
**Pull request by DragosDumitrache**
Updating installation instruction for Gradle 4 and 5

### CODE REVIEW

File: content/installation/java.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ You can now run Cucumber [from the command line](/cucumber/api/#from-the-command
 If you are going to use the lambda expressions API (Java 8) to write the step
 definitions, add the following dependencies to `build.gradle`:
 
+## Gradle 4.10.3 or older
```
Comment (mlvandijk) [inline content/installation/java.md]: Maybe also don't use headers here, but versions in **bold**, consistent with your other PR (#317)

**Review by mlvandijk — COMMENTED** [review]
Thanks for this @DragosDumitrache! Small change request to make it consistent with the other PR if you don't mind..

File: content/installation/java.md (hunk 1 above)
Comment (DragosDumitrache) [reply content/installation/java.md]: Sure thing. I only thought about using headers as it was easier to quickly navigate to the appropriate instructions based on the Gradle versions one uses from a table of contents, or similar.

I've also had a look at other places where updates might be required, but nothing immediately apparent

**Review by mlvandijk — APPROVED** [review]

**Comment by mlvandijk** [comment]
Thanks again @DragosDumitrache!

**Pull request merged**